bin/gbuild

```diff
@@ -163,8 +163,10 @@ EOF"
     build_desc["remotes"].each do |remote|
       dir = sanitize(remote["dir"], remote["dir"])
 
-      author_date = `cd inputs/#{dir} && TZ=UTC git log --date='format-local:%F %T' --format="%ad" -1`.strip
-      raise "error looking up author date in #{dir}" unless $?.exitstatus == 0
+      Dir.chdir("inputs/#{dir}") do
+        author_date = `TZ=UTC git log --date='format-local:%F %T' --format="%ad" -1`.strip
+        aise "error looking up author date in #{dir}" unless $?.exitstatus == 0
```

Comment:
`aise`

Reply:
:facepalm: good catch. I've fixed this and force-pushed the change to the PR.